src/codegen/data_table_proxy.cpp

```diff
@@ -38,17 +39,40 @@ llvm::Type* DataTableProxy::GetType(CodeGen& codegen) {
 }
 
 const std::string& DataTableProxy::_GetTileGroupCount::GetFunctionName() {
+  static const std::string kGetTileGroupCount =
+      "_ZNK7peloton7storage9DataTable17GetTileGroupCountEv";
```

Comment:
ifdef __APPLE__ is missing.  Although this macro is not used at the moment, it would be a good practice to keep it as all the codegen code currently have these.  Consistency is much better if we work together on a large source code base.